backends/dpdk/dpdkArch.cpp

```diff
@@ -1265,8 +1268,23 @@ const IR::Node* SplitP4TableCommon::postorder(IR::MethodCallStatement *statement
         if (match_tables.count(table->name) == 0)
             return statement;
         auto decls = new IR::IndexedVector<IR::StatOrDecl>();
-        decls->push_back(statement);
         auto tableName = apply->object->getName().name;
+
+        // member_id and/or group_id must be initialized prior to member table apply.
+        if (member_ids.count(tableName) != 0) {
+            auto member_id = member_ids.at(tableName);
+            decls->push_back(new IR::AssignmentStatement(
+                        new IR::PathExpression(member_id),
+                        new IR::Constant(IR::Type_Bits::get(32), 0))); }
+
+        if (group_ids.count(tableName) != 0) {
+            auto group_id = group_ids.at(tableName);
+            decls->push_back(new IR::AssignmentStatement(
+                        new IR::PathExpression(group_id),
+                        new IR::Constant(IR::Type_Bits::get(32), 0))); }
```

Comment:
always 0?

Reply:
always initialized to zero, a subsequent table will set it based on control plane information.